scripts/data/format_args.py

```diff
@@ -110,14 +110,18 @@ def format_item(item):
     return format_item(args)
 
 
-def format_args(input_file, execute_script):
+def format_args(input_file, execute_script, cairo1_run):
     """Reads arguments from JSON file and prints formatted result.
     Expects a single CLI argument containing file path.
     Output is compatible with the Scarb runner arguments format.
     """
     args = json.loads(Path(input_file).read_text())
     res = flatten_tuples(serialize(args))
-    print([res, 1 if execute_script else 0])
+    flag = 1 if execute_script else 0
+    if cairo1_run:
+        print(f"{format_cairo1_run(res)} {flag}")
```

Comment:
What is `format_cairo1_run`?

Reply:
cairo1-run uses a different arguments format vs scarb
format_cairo1_run is a function that was always there, just not used :)